src/main/java/uk/gov/companieshouse/web/pps/service/finance/impl/FinanceServiceHealthCheckImpl.java

```diff
@@ -114,11 +117,14 @@ private PPSServiceResponse getRedirectPath(FinanceHealthcheck financeHealthcheck
     }
 
     private Optional<String> getParsedDateTime(final String endTime) {
-        DateFormat inputDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX");
-        DateFormat displayDateFormat = new SimpleDateFormat("h:mm a z 'on' EEEE d MMMM yyyy");
+
         try {
-            return Optional.of(displayDateFormat.format(inputDateFormat.parse(endTime)));
-        } catch (ParseException ex) {
+            Instant instant = Instant.parse(endTime);
+            ZonedDateTime displayDateFormat = instant.atZone(ZoneId.of("Europe/London"));
+            DateTimeFormatter inputDateFormat = DateTimeFormatter.ofPattern("h:mm a 'on' EEEE d MMMM yyyy", Locale.UK);
+
+            return Optional.of(displayDateFormat.format(inputDateFormat));
```

Comment:
Following on from my other comment around the server offset.
I'm wondering whether it would be better to do this instead?
```suggestion
            return Optional.of(OffsetDateTime.parse(endTime)
                    .atZoneSameInstant(ZoneId.of("Europe/London"))
                    .format(inputDateFormat));
```

Reply:
Yep, should be fine to change this.